src/bijectors/truncated.jl

```diff
@@ -68,6 +68,27 @@ end
 
 with_logabsdet_jacobian(b::TruncatedBijector, x) = transform(b, x), logabsdetjac(b, x)
 
+# from stan https://github.com/stan-dev/math/blob/develop/stan/math/prim/constraint/lub_constrain.hpp etc
+function truncated_inv_logabsdetjac(x, a, b)
+    lowerbounded, upperbounded = isfinite(a), isfinite(b)
+    if lowerbounded && upperbounded
+        return log(b - a) - abs(x) + 2.0 * LogExpFunctions.log1pexp(-abs(x))
```

Comment:
Generally, one should avoid `Float64` literals since they can cause undesired promotions (e.g. for `Float32` arguments):
```suggestion
        abs_y = abs(y)
        return log(b - a) - abs_y + 2 * LogExpFunctions.log1pexp(-abs_y)
```